client/src/components/Common/ErrorPluginTracker.vue

```diff
@@ -0,0 +1,16 @@
+<script setup lang="ts">
+import { getGalaxyInstance } from "@/app";
+import { computed } from "vue";
+
+const errorReportingAvailable = computed(() =>{
+    const Galaxy = getGalaxyInstance();
+    if (!!Galaxy.Sentry.isInitialized) {
+        return "Your error has been logged in Sentry to improve your experience.";
```

Comment:
I wouldn't explicitly mention Sentry, users may not know what it is. Maybe something like:
```suggestion
        return "The error has been logged in our issue tracker system, and our team is investigating to fix it as soon as possible.";
```
Or something similar :)